<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showUserReservations</class>
 <widget class="QDialog" name="showUserReservations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>690</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>1031</width>
     <height>501</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Room name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start day</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cancel</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Start date filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_start_day">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>87</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_day_filter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="room_filter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Room name filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>End date filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_end_day">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>87</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="end_date_filter">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
